src/aap_eda/core/management/commands/rotate_db_encryption_key.py

```
@@ -0,0 +1,154 @@
+        rollback, matching ``awx-manage regenerate_secret_key``
+        behaviour.
+        """
+        total = 0
+        for model, field in fields:
+            table = model._meta.db_table
+            col = field.column
+            pk_col = model._meta.pk.column
+            with connection.cursor() as cur:
+                cur.execute(
+                    f'SELECT "{pk_col}", "{col}" FROM "{table}" '
+                    f'WHERE "{col}" IS NOT NULL',
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)